docs/tutorials/ruby-driver-create-client.txt

```diff
@@ -97,8 +97,20 @@ URI Options Conversions
    * - URI Option
      - Ruby Option
 
-   * - replicaSet=String
-     - ``:replica_set => String``
+   * - appName=String
+     - ``:app_name => String``
+
+   * - authMechanism=String
+     - ``:auth_mech => Symbol``
+
+   * - authMechanismProperties=Strings
```

Comment:
How would a user determine what to put in `Strings` for this option?

Reply:
This already existed in the old code; I just alphabetized the list for easier reading. If you just look at the diff for https://github.com/mongodb/mongo-ruby-driver/pull/1170/commits/e396935447b59a9cf05b2622cb579227623ebdcb, you'll see the options that are new.

Reply:
I've added extra documentation to this option and the others below where the type being forwarded is different from the type specified in the URI. Let me know if you think any of the other options need extra documentation as well.